dashboard/config/routes.rb

```diff
@@ -317,6 +317,13 @@ def redirect_to_teacher_dashboard
 
   post '/plc/course_units/:id/submit_new_questions_and_answers', to: 'plc/course_units#submit_new_questions_and_answers'
 
+  namespace :pd do
+    get 'workshops/:workshop_id/enroll', action: 'new', controller: 'workshop_enrollment'
+    post 'workshops/:workshop_id/enroll', action: 'create', controller: 'workshop_enrollment'
+    get 'workshop_enrollment/:code', action: 'show', controller: 'workshop_enrollment'
```

Comment:
Oh I see, that makes a bunch of sense.  No strong opinions on this.  I guess in this case we have two nouns - works and workshop_enrollment - and those actions are appropriate for each.  Are we comfortable with the security implications around these enrollments not being protected by an account?